Description: Use libgetdns for DNS resolution
Author: Felix Lechner <felix.lechner@gmail.com>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=760467
Forwarded: no
Last-Update: 2014-10-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -146,6 +146,8 @@ AC_SUBST(LESS_STRICT_OPTIONS)
 
 # Checks for libraries.
 
+AC_SEARCH_LIBS([getdns_address], [getdns])
+
 # Checks for header files.
 AC_ARG_WITH( antlr,
       [  --with-antlr      Set prefix where libantlr3c can be found (ex:/usr or /usr/local)[default=PREFIX] ],
--- a/src/belle_sip_resolver.c
+++ b/src/belle_sip_resolver.c
@@ -17,7 +17,6 @@
 */
 
 #include "belle_sip_internal.h"
-#include "dns.h"
 
 #include <stdlib.h>
 #ifdef __APPLE__
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -38,7 +38,6 @@ libbellesip_la_SOURCES=	\
 			transports/tls_listeningpoint_polarssl.c \
 			transports/tls_channel_polarssl.c \
 			refresher.c \
-			dns.c dns.h \
 			belle_sip_dict.c \
 			generic-uri.c \
 			http-provider.c \
